README.md

```diff
@@ -41,12 +41,18 @@ private
 def client
   @client ||= Dor::Services::Client.configure(url: Settings.dor_services.url,
                                               username: Settings.dor_services.user,
-                                              password: Settings.dor_services.pass)
+                                              password: Settings.dor_services.pass,
```

Comment:
remove password entirely, it's not used.

Reply:
Done.